UtilizeSkylineDataServices/Readme.md

```diff
@@ -23,104 +23,145 @@ From the panel of the web app, you will be able to:
 
-However, when the web application is hosted by SystemLink Cloud, you don’t need an API key or a server URL because your **ni.com credentials** are used instead. This deliberately minimizes security vulnerabilities that can arise when using an API key on your hosted web application.
+![Configure GVI in LabVIEW vs Hosting in SystemLink Cloud](https://ni.github.io/webvi-examples/UtilizeSkylineDataServices/cloud-config.PNG)
 
-## What is an API Key?
+_Figure: Note the input differentiation between connecting to the SystemLink Cloud server from LabVIEW or LabVIEW NXG 2.1 Web Module vs. hosting your web application on SystemLink Cloud._
 
-An API key authenticates an application, device, or system – much like your ni.com credentials authenticate who you are. Assigning an API key to an application allows it to connect to the SystemLink Cloud.
+## What is an API Key?
+An API key authenticates an application trying to access SystemLink Cloud. It helps to think of an API key like login credentials. When you log in somewhere, your username and password authenticate who you are and, if they’re correct, grant you access. An API key functions similarly for your application connecting to the SystemLink Cloud server.
 
 ## Obtaining an API Key
-1.	Go to SystemLink Cloud.com and login with your **ni.com credentials**.
-2.	Click the **Security** button.
-3.	Click **+ New API key** to create an API key.
-4.	Click **Copy to clipboard** to save the API key.
-> **Note**:  You only get to see an API key once, so retain it somewhere safe and only provide it to those you trust. If you delete an API key, all applications using that API key will no longer be able to connect to SystemLink Cloud.
-
-5. Click **Create** at the bottom of the dialog box to activate the API key.
-> **Note**: If you do not click **Create**, you will not create the API key. Therefore, the API key you copied will be invalid.
-
-## Updating Your Skyline API Keys
-In order to connect your systems to Helium, you need to update or install the Skyline API keys. Depending on your version of LabVIEW, follow the instructions below.
-
-### LabVIEW NXG:
-1.	Go to the SystemLink Cloud.com and click the **Data Services APIs** button.
-2.	Click the **Download Skyline API Key Support** button.
-3.	Double click the package file to open **NI Package Manager**.
-4.	Follow instructions in the installation prompt to complete installation.
+1.	Go to [Security](https://www.systemlinkcloud.com/security).
+2.	Click **+ NEW API KEY** to create an API key.
+3.	Click **Copy key** to save the API key.
+>**Note:** You only get to see an API key once, so keep it somewhere safe and only provide it to those you trust. If you delete an API key, all applications using that API key will no longer be able to connect to SystemLink Cloud.
+
+4.	Click **Update** at the bottom of the dialog box to activate the API key.
```

Comment:
This is no longer true. The API Key is valid and usable as soon as the user sees the API key. They do not need to click the update button anymore.